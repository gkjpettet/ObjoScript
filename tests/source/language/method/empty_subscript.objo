var list = [1, 2]
list[] // expect error