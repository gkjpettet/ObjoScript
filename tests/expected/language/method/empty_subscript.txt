At least one subscript index is required.